src/utl/include/utl/Logger.h

```diff
@@ -168,7 +171,7 @@ class Logger
   // API as we are writing JSON not user messages.
   // Note: these methods do no escaping so avoid special characters.
   template <typename T,
-            typename = std::enable_if_t<std::is_arithmetic_v<T>>>
+            std::enable_if_t<std::is_arithmetic<T>::value>* = nullptr>
```

Comment:
Why this change?

Reply:
The previous version did not build once the non-string `metric` was invoked. 
I think because `enable_if_t` expects a boolean from `::value` instead of the `is_arithmetic` struct itself and `enable_if_t` itself will be a `void` type if the condition is met.

Reply:
It can just be changed to:

typename = std::enable_if_t<std::is_arithmetic_v<T>**::value**>